packages/dev/docs/pages/releases/2022-05-03.mdx

```diff
@@ -0,0 +1,224 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: DatePicker and Calendar are now in RC, complete with new documentation! Other updates include bug and accessibility fixes for our existing components. Thank you to all who contributed to this release!
+date: 2022-05-03
+---
+
+# May 3, 2022 Release
+We are pleased to announce that the `datepicker` and `calendar` packages have been release into rc! This update comes complete with new documentation for the [DatePicker](https://react-spectrum.adobe.com/react-spectrum/DatePicker.html), [DateRangePicker](https://react-spectrum.adobe.com/react-spectrum/DateRangePicker.html), [DateField](https://react-spectrum.adobe.com/react-spectrum/DateField.html), [TimeField](https://react-spectrum.adobe.com/react-spectrum/TimeField.html), [Calendar](https://react-spectrum.adobe.com/react-spectrum/Calendar.html) and [RangeCalendar](https://react-spectrum.adobe.com/react-spectrum/RangeCalendar.html) components and associated [React-Aria](https://react-spectrum.adobe.com/react-aria/useDatePicker.html) hooks.
+
+Other updates include bug and accessibility fixes for our existing components. Thank you to all who contributed to this release!
+
+
+## Fixes
+- Fix `useNumberField` to propagate name/placeholder props - [@MeowIng44](https://github.com/MeowIng44) - [PR](https://github.com/adobe/react-spectrum/pull/2884)
+- `useSelect` should call `onFocusChange` when focus is changed - [@lishichengyan](https://github.com/lishichengyan) - [PR](https://github.com/adobe/react-spectrum/pull/2970)
+- Adjust `Dialog` to better position footers, buttons, and headers - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/2820)
+- Prevent `focusedKey` from scrolling into view on click/press - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2725)
+- Deprecate `placeholder` prop on most form components - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/2966)
```

Comment:
Spectrum form components?

just to make it clear we didn't deprecate for the hooks